packages/k8s/k3s-openrc/build.yaml

```diff
@@ -0,0 +1,32 @@
+requires:
```

Comment:
you dont need another build file, you can use templating to generate one package or the other in the same build file!

Reply:
They are quite different on the way they fake openrc/systemd, the files they copy and such. We would remove duplication but introduce templating complexity. I'd rather keep them separate for now. What do you think?

Reply:
Good to me!

Reply:
Nevermind, we tried the templating and it turns out it doesn't look too complicated. We are going with your suggestion.